.github/workflows/release-version.yml

```diff
@@ -0,0 +1,112 @@
+          NEW_VERSION_TAG="v$NEW_VERSION" # e.g. v1.1.0
+
+          # Update README with new version
+          sed -i "s/$OLD_VERSION/$NEW_VERSION/" {README.md,src/examples/*.yml}
+
+          echo "NEW_VERSION_TAG=$NEW_VERSION_TAG" >> $GITHUB_OUTPUT
+      - name: Commit new version
+        run: |
+          git add --all
+          git commit -m ${{ steps.bump-version.outputs.NEW_VERSION_TAG }}
+      - name: Push the branch
+        run: git push -u origin local-branch:release/${{ steps.bump-version.outputs.NEW_VERSION_TAG }}
+
+      # Create the pull request
+      - name: Generate release notes
+        id: generate-release-notes
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ steps.get-token.outputs.token }}
+          script: |
+            const { data: releaseNotes } = await github.rest.repos.generateReleaseNotes({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              tag_name: '${{ steps.bump-version.outputs.NEW_VERSION_TAG }}',
+            })
+
+            core.setOutput('RELEASE_NOTES', releaseNotes.body)
+      - name: Create pull request
+        id: create-pull-request
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ steps.get-token.outputs.token }}
+          script: |
+            const { data: pullRequest } = await github.rest.pulls.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              base: 'main',
+              head: 'release/${{ steps.bump-version.outputs.NEW_VERSION_TAG }}',
+              title: '[release:${{ github.event.inputs.semver }}] `${{ steps.bump-version.outputs.NEW_VERSION_TAG }}`',
+              body: ${{ toJSON(steps.generate-release-notes.outputs.RELEASE_NOTES) }}
+            })
+
+            core.setOutput('PULL_REQUEST_NUMBER', pullRequest.number)
+      - name: Create comment
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ steps.get-token.outputs.token }}
+          script: |
+            const body = `Once merged, this PR will automatically create a GitHub release for you.
+            The description of the release will exactly match this PR's description. Feel free to edit it.
+
+            > [!WARNING]
+            > Publishing the development and production orbs will fail because the creator of the tag is the [Release Automation](https://github.com/apps/ci-integrations-release-automation) github app, which is seen as a bot by CircleCI.
+            >
+            > Specifically, the \`lint-pack\` workflow will fail first, then once passed - the \`test-deploy\` workflow will also fail.
+            > You should look for workflows pointing to the \`${{ steps.bump-version.outputs.NEW_VERSION_TAG }}\` tag.
+            > To make them pass, please \`Rerun the workflow from start\` [here](https://app.circleci.com/pipelines/github/DataDog/synthetics-test-automation-circleci-orb) on CircleCI.`
```

Comment:
Can you change the warning's text to be something along the lines of "You now have to create a PR on https://github.com/bitrise-io/bitrise-steplib via [our Datadog fork](https://github.com/DataDog/bitrise-steplib) to officially publish the Bitrise step."

Reply:
I was thinking for this should we not raise the PR to the fork automatically 🤔 
Although i'm not sure it's possible as i see that the PR here are created from this repo and i don't think we would be able to do the same from the fork 🤔

Reply:
This would bring permission difficulties, which are unnecessary for now. Let's focus on the release automation alone 👌